channeldb/graph.go

```diff
@@ -3186,6 +3190,9 @@ func putChanEdgePolicy(edges, nodes *bbolt.Bucket, edge *ChannelEdgePolicy,
 	if err := binary.Write(&b, byteOrder, uint64(edge.MinHTLC)); err != nil {
 		return err
 	}
+	if err := binary.Write(&b, byteOrder, uint64(edge.MaxHTLC)); err != nil {
```

Comment:
This won't work, as it will not be a backwards compatible db change. Instead, care must be taken to write it last, as part of what was previously opaque data :)

Reply:
thanks :) fixed